packages/dapi-grpc/Cargo.toml

```diff
@@ -35,7 +35,7 @@ mocks = ["serde", "dep:serde_json"]
 [dependencies]
 prost = { version = "0.13" }
 futures-core = "0.3.30"
-tonic = { version = "0.12", features = [
+tonic = { version = "0.12.3", features = [
```

Comment:
it must be =0.12.3 isn't it?

Reply:
No, we allow upgrades, `Patched >=0.12.3`  https://rustsec.org/advisories/RUSTSEC-2024-0376.html